tests/ui_tests/test_service_page.py

```diff
@@ -248,6 +265,96 @@ def test_save_custom_config_on_service_config_page(self, app_fs, sdk_client_fs):
         service_config_page.config.save_config()
         service_config_page.config.compare_versions(params["config_name_old"])
         service_config_page.config.check_config_fields_history_with_test_values()
+        with allure.step("Check invisible params"):
+            config = service.config()
+            assert len(config.keys()) == 28, "There are should be 24 config parameters"
+            for group in self.INVISIBLE_GROUPS:
+                assert group in config.keys(), "Invisible group should be present in config object"
+
+    def test_save_advanced_config_on_service_config_page(self, app_fs, sdk_client_fs):
+        """Test config save with advanced params on /cluster/{}/service/{}/config page"""
+
+        params = {
+            "config_name_new": "test_name",
+            "config_name_old": "init",
+        }
+        with allure.step("Create cluster and service"):
+            bundle = cluster_bundle(sdk_client_fs, "service_with_advanced_params")
+            cluster = bundle.cluster_create(name=CLUSTER_NAME)
+            service = cluster.service_add(name=SERVICE_NAME)
+        service_config_page = ServiceConfigPage(app_fs.driver, app_fs.adcm.url, cluster.id, service.id).open()
+        with allure.step("Check that rows are invisible"):
+            config_rows = service_config_page.config.get_all_config_rows()
+            assert len(config_rows) == 0, "Rows should be hidden"
+        service_config_page.config.click_on_advanced()
+        with allure.step("Check that rows are visible"):
+            config_rows = service_config_page.config.get_all_config_rows()
+            assert len(config_rows) == 16, "Rows should be visible"
```

Comment:
Maybe specify how many rows do you expect, so it won't be confusing when on the screenshot there'll be 2 rows instead of 16?

Reply:
added const